lib/modules/manager/pep621/processors/uv.ts

```diff
@@ -148,20 +154,6 @@ export class UvProcessor implements PyProjectProcessor {
     return Promise.resolve(deps);
   }
 
-  async getLockfiles(
-    _project: PyProject,
-    lockfiles: string[],
-    packageFile: string,
-  ): Promise<string[]> {
-    const lockFileName = await findLocalSiblingOrParent(packageFile, 'uv.lock');
```

Comment:
we need to override because base class function isn't using the find local sibling or parent. UV supports workspaces with multiple projects and single lock file like pnpm...

Reply:
hmm, I switched to use `findLocalSiblingOrParent` by default. I think this should be fine as this can only lead to issues if people would nest multiple projects inside each other and fail to commit the lockfile 🤔